app/models/post.rb

```diff
@@ -0,0 +1,24 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+  scope :user, -> { joins(:users).where(user: { user_id: id }) }
+  scope :comments, -> { joins(:comments).where(comments: { post_id: id }) }
+  scope :likes, -> { joins(:likes).where(likes: { post_id: id }) }
+
+  def update_post_counter
+    user.update(post_counter: Post.where(user_id: user.id).count)
+  end
+
+  def last_5_comments
+    comments.limit(5)
+  end
```

Comment:
- [ ] Just as mentioned in the `User` model, for this method you need to take the 5 most recent comments on the post, so again, you would need to use the `created_at` property. Please work on this issue.